segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/metadata/Constants.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.metadata;
+
+/**
+ * Class that defines various constants.
+ */
+public final class Constants {
```

Comment:
I would not be making these public. Please make sure that any `setFlag`/`getFlag` method is private to the class(es) they are defined in and you expose proper methods to set them. Nobody likes to deal with flags and masks.

Reply:
need to use in two package.